website/content/blog/2023-11-reduce-reuse-rebase-buildpacks.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Reduce, Reuse, Rebase: Sustainable Containers with Buildpacks"
+description: Buildpacks are the greenest choice for container builds
```

Comment:
I would like to remove claims like "the greenest choice."  Such assertions should not be done without measurements, and there may be other "greener" container builds.

Reply:
I suggested some different language in a comment below

Reply:
Fixed in e0aea31